bundles/org.openhab.binding.ferroamp/src/main/java/org/openhab/binding/ferroamp/internal/EsmParameters.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.openhab.binding.ferroamp.internal;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link EsmParameters} is responsible for all parameters regarded to ESM
+ *
+ * @author Örjan Backsell - Initial contribution
+ *
+ */
+
+@NonNullByDefault
+public class EsmParameters {
+    public String jsonPostEsm;
+
+    public EsmParameters(String jsonPostEsm) {
+        this.jsonPostEsm = jsonPostEsm;
+    }
+
+    public static List<String> getChannelParametersEsm() {
```

Comment:
1. This method does not seem to be used anywhere.
2. Use the defined FerroampBindingConstants where possible.

Reply:
20241107
Deleted this class, it's not used anymore

Reply:
Still there.

Reply:
20241111 Deleted

Reply:
Still there.

I'm not sure about the used workflow, somehow we keep doing this in circles. Please verify that the changes you make are actually in the commit and pushed to Github.

Reply:
They are gone now.